systemtest/src/test/java/io/strimzi/systemtest/upgrade/StrimziUpgradeST.java

```diff
@@ -176,18 +182,16 @@ void testChainUpgrade() throws Exception {
     void testUpgradeKafkaWithoutVersion() throws IOException {
         File dir = FileUtils.downloadAndUnzip(latestReleasedOperator);
 
-        coDir = new File(dir, "strimzi-0.19.0/install/cluster-operator/");
+        File kafkaPersistent = new File(dir, "strimzi-" + latestReleasedVersion + "/examples/kafka/kafka-persistent.yaml");
+        coDir = new File(dir, "strimzi-" + latestReleasedVersion + "/install/cluster-operator/");
 
         // Modify + apply installation files
         copyModifyApply(coDir);
+        // Apply Kafka Persistent without version
+        applyKafkaWithoutVersion(kafkaPersistent);
```

Comment:
Should we also add `CLUSTER_NAME` as a parameter and change it in the `kafkaPersistent`? It is "hardcoded" in the yaml to be `my-cluster` but in the ST it might be changed eventually.

Reply:
I think that's a good idea! Thanks!